src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,46 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 9;
+    private int size;
+    private Object[] keys;
+    private Object[] values;
```

Comment:
```suggestion
    private K[] keys;
    private V[] values;
```
if you do this, you won't need to casting